packages/peregrine/lib/talons/Checkout/useForm.js

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useState } from 'react';
+import { useQuery } from '@apollo/react-hooks';
+
+export const useForm = props => {
+    const { countriesQuery } = props;
+
+    const [countries, setCountries] = useState(null);
+
+    const { data: countriesData } = useQuery(countriesQuery);
+
+    useEffect(() => {
+        if (countriesData) {
+            setCountries(countriesData.countries);
```

Comment:
Do you need state/effect for this? I think you can just return `countriesData` right?

Reply:
You da man: https://github.com/magento/pwa-studio/pull/1993/commits/5cbc333c2dce7c892d75927754186d8c6dd12dd5.